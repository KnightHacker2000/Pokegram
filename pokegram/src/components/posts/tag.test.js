/**
 * @jest-environment jsdom
 */
import React from 'react';
//  import { fireEvent, render } from '@testing-library/react';
import ShallowRenderer from 'react-test-renderer/shallow';
import TagPhoto from './Tag';
import '@testing-library/jest-dom/extend-expect';
// import { test } from '@jest/globals';

test('test tagging snapshot', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<TagPhoto PID={2} handleTagState={()=> {}} />);
  const tagComp = renderer.getRenderOutput();
  // expect(tree).toMatchSnapshot();
});

// test('test profile click', () => {
//   const renderer = new ShallowRenderer();
//   renderer.render(<ResponsiveAppBar homeStates={parentStates} />);
//   const res = renderer.getRenderOutput();
//   // const sha = shallow()
//   // const tree = menuComp.toJSON();
//   res.props.children.props.children.props.children[0].props.onClick();
//   // expect(tree).toMatchSnapshot();
// });

// test('test follow button', () => {
//   const renderer = new ShallowRenderer();
//   renderer.render(<Profile homeStates={parentStates} />);
//   const res = renderer.getRenderOutput();
//   res.props.children[0].props.children.props.children[1].props.children[0].props.children[1].props.onClick({ preventDefault: () => {} });
// });