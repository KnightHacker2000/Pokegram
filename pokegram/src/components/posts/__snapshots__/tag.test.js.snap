// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test tagging snapshot 1`] = `
<div
  style={
    {
      "alignItems": "center",
      "background": "rgba(90, 90, 90, 0.8)",
      "bottom": 0,
      "display": "flex",
      "flexDirection": "row",
      "left": 0,
      "position": "fixed",
      "right": 0,
      "top": 0,
      "zIndex": 99,
    }
  }
>
  <div
    style={
      {
        "alignItems": "center",
        "background": "white",
        "border": "1px solid rgba(0, 0, 0, 0.05)",
        "borderRadius": "10px",
        "display": "flex",
        "height": "max-content",
        "marginBottom": "auto",
        "marginLeft": "auto",
        "marginRight": "auto",
        "marginTop": "auto",
        "zIndex": 99,
      }
    }
  >
    <ForwardRef(Container)
      maxWidth="xs"
    >
      <ForwardRef(Box)
        sx={
          {
            "alignItems": "center",
            "display": "flex",
          }
        }
      >
        <Memo
          onClick={[Function]}
          sx={
            {
              "width": "200%",
            }
          }
        />
      </ForwardRef(Box)>
      <ForwardRef(List)
        sx={
          {
            "bgcolor": "background.paper",
            "maxWidth": 360,
            "width": "100%",
          }
        }
      >
        <p>
          NO Tagging
        </p>
      </ForwardRef(List)>
    </ForwardRef(Container)>
  </div>
</div>
`;
